instrumentation/opentelemetry_tesla/lib/middleware/opentelemetry_tesla_middleware.ex

```diff
@@ -86,14 +91,14 @@ defmodule Tesla.Middleware.OpenTelemetry do
     url = Tesla.build_url(url, query)
     uri = URI.parse(url)
 
-    attrs = %{
-      "http.method": http_method(method),
-      "http.url": url,
-      "http.target": uri.path,
-      "http.host": uri.host,
-      "http.scheme": uri.scheme,
-      "http.status_code": status_code
-    }
+    attrs =
+      %{}
```

Comment:
Should keep this as a single map definition instead of piping through a bunch of `put`s.

Reply:
I was going to ask about this. I think the only way to not do that is to use arrow style maps, but that results in a different structure for matching against. Am I missing something?

Reply:
I tried. Any idea how it would be?

Reply:
I don't think it results in anything different:

```
%{Trace.http_method() => http_method(method)
```

Right?

Reply:
It's fine. It's just some tests that might need to be adjusted anytime there's a case where users can merge in their own attributes once we implement that in all of these.

Reply:
ahhh the rocket syntax ...!!